bin/rucio

```diff
@@ -2766,6 +2775,7 @@ Commands:
     info_rule_parser.set_defaults(which='info_rule')
     info_rule_parser.add_argument(dest='rule_id', action='store', help='The rule ID')
     info_rule_parser.add_argument('--examine', dest='examine', action='store_true', help='Detailed analysis of transfer errors')
+    info_rule_parser.add_argument('--include-ttc', dest='get_ttc', action='store_true', help='Show Estimated Time To Complete for the rule. Calculation can be time consuming.')
```

Comment:
Change `get_ttc` to `estimate_ttc`

Reply:
You mean the name of the variable or the name of the argument? The command will be invoked as
rucio rule-info --include-ttc 12abc..1234abc
and you wanted to change it to 
rucio rule-info --get-ttc 12abc..1234abc
?

Reply:
--estimate-ttc

Reply:
Done, yet I prefer --include-ttc :P